scripts/generate_site_sidebar/sidebar_template.md

```diff
@@ -9,7 +9,7 @@
         - [update](reference/pkg/update/)
     - [fn](reference/fn/)
         - [render](reference/fn/render/)
-        - [run](reference/fn/run/)
+        - [eval](reference/fn/eval/)
         - [sink](reference/fn/sink/)
```

Comment:
as a follow up, sink and source also be updated (they're stale and have broken links)